host-configs/rzansel-blueos_3_ppc64le_ib_p9-xl@coral.cmake

```diff
@@ -16,20 +16,20 @@
 # Compilers
 #------------------------------------------------------------------------------
 
-set(CMAKE_C_COMPILER "/usr/tce/packages/xl/xl-2019.12.23/bin/xlc" CACHE PATH "")
+set(CMAKE_C_COMPILER "/usr/tce/packages/xl/xl-2019.08.20/bin/xlc" CACHE PATH "")
```

Comment:
@white238 -- for some reason the ``xl@coral`` spec used the new ``xl@coral_nvcc`` compiler.

Looking into it...

@tgamblin -- this is likely on the ``spack`` side.  Is this expected?  See the changes to ``compilers.yaml`` below.

Reply:
A simple resolution would likely be to add a new suffix to ``xl@coral``.
Should I do that?

Reply:
I think part of the problem is that we're using some parts (but not all) of the ``spec`` to name the generated ``host-config`` file.

Reply:
My guess is that ``spack`` ignores the part after the underscore and the ``xl@coral_nvcc`` was overwriting ``xl@coral``.

I resolved this by naming the new compiler spec ``xl@nvcc`` and keeping the old one as ``xl@coral``.